c2cgeoportal/scaffolds/create/print/print-apps/+package+/config.yaml.mako

```diff
@@ -55,7 +55,7 @@ templates:
             - !restrictUris
                 matchers:
                 - !localMatch
-                  pathRegex: "^/${instanceid}/wsgi/mapserv_proxy$"
+                  pathRegex: "/${instanceid}/wsgi/mapserv_proxy"
```

Comment:
I'm not sure about that...

The print is doing that:
```java
Matcher matcher = Pattern.compile(this.pathRegex).matcher(matchInfo.getPath());
if (!matcher.matches()) {
```
That doesn't imply the `^` and the `$`...